src/pages/news/NewsList.tsx

```diff
@@ -0,0 +1,75 @@
+import { uuid } from 'uuidv4'
+
+import SingleNews from '@/pages/news/SingleNews'
+
+const NewsList = () => {
+  const newsList = [
```

Comment:
For this kind of harcoding data cases put TODO: `~` Comment

Reply:
Done